feature/google/src/main/java/com/revenuecat/purchases/google/storeProductConversions.kt

```diff
@@ -13,20 +13,23 @@ fun ProductDetails.toStoreProduct(
     offerDetails: List<ProductDetails.SubscriptionOfferDetails>
 ): GoogleStoreProduct {
     val subscriptionOptions = offerDetails.map { it.toSubscriptionOption(productId, this) }
-    val bestOffer = subscriptionOptions.findBestOffer()
+    val defaultOffer = subscriptionOptions.findDefaultOffer()
+    val basePlanPrice = subscriptionOptions.firstOrNull { it.isBasePlan }?.recurringPhase?.price
+
+    subscriptionOptions.first { it.pricingPhases.dropLast(1).any { phase -> phase.price.amountMicros == 0L } }
 
     return GoogleStoreProduct(
         productId,
         productType.toRevenueCatProductType(),
-        createOneTimeProductPrice(),
+        createOneTimeProductPrice() ?: basePlanPrice,
         title,
         description,
         offerDetails.firstOrNull { it.isBasePlan }?.subscriptionBillingPeriod,
         offerDetails.map { it.toSubscriptionOption(
```

Comment:
not your code but realized while reviewing -- we should reuse `subscriptionOptions` here instead of re-calculating

Reply:
Oh, I think this _might_ have been my code from when I worked best off 😛 Whooooppss